src/info.js

```diff
@@ -179,15 +179,17 @@ function renderDeprecation(into, options, def) {
   }
 }
 
-function text(into, content, className, options, ref) {
+function text(into, content, className, options = {onClick: null}, ref) {
   if (className) {
     const onClick = options.onClick;
-    const node = document.createElement(onClick ? 'a' : 'span');
+    let node = document.createElement('span');
     if (onClick) {
+      node = document.createElement('a');
```

Comment:
I prefer the old const code, because it’s const over let, and because it requires less garbage collection (not creating a node we’re going to immediately throw away). I’m guessing this was to solve a Flow issue, can we assign the span in the else branch instead? Or maybe moving the ternary to the root of the assignment (rather than the arg to createElement) would solve it? If neither of these work, please add a comment explaining why the code seems non-optimal 👍

Reply:
I will submit a revision without using `let`, no problem!  I'll just need to branch the logic entirely

Reply:
We can't use the ternary because `span` doesn't have `href`. That was the original flow issue.